packages/valory/skills/hello_world_abci/rounds.py

```diff
@@ -80,11 +81,10 @@ def synchronized_data(self) -> SynchronizedData:
         return cast(SynchronizedData, self._synchronized_data)
 
 
-class RegistrationRound(CollectDifferentUntilAllRound, HelloWorldABCIAbstractRound):
+class RegistrationRound(CollectSameUntilAllRound, HelloWorldABCIAbstractRound):
```

Comment:
This was wrong and only detected due to the change. The `sender` does not count towards evaluating sameness of payloads.

Reply:
here is another: https://github.com/valory-xyz/open-autonomy/blob/8b725f975804c146edabf4afc911836034559343/docs/key_concepts/abci_app_abstract_round.md?plain=1#L35

Reply:
Ah good find!